src/main/java/uk/gov/ons/census/fwmt/jobservice/transition/EmptyCreateUnitCancel.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ons.census.fwmt.jobservice.transition;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.fwmt.common.rm.dto.ActionInstructionType;
+import uk.gov.ons.census.fwmt.common.rm.dto.FwmtCancelActionInstruction;
+import uk.gov.ons.census.fwmt.events.component.GatewayEventManager;
+import uk.gov.ons.census.fwmt.jobservice.data.GatewayCache;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.InboundProcessor;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.ProcessorKey;
+
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_CANCEL_PRE_SENDING;
+
+@Qualifier("Cancel")
+@Component
+public class EmptyCreateUnitCancel implements InboundProcessor<FwmtCancelActionInstruction> {
+
+  @Autowired
+  private GatewayEventManager eventManager;
+
+  private static ProcessorKey key = ProcessorKey.builder()
+      .actionInstruction(ActionInstructionType.CANCEL.toString())
+      .surveyName("CENSUS")
+      .addressType("CE")
+      .addressLevel("U")
+      .build();
+
+  @Override
+  public ProcessorKey getKey() {
+    return key;
+  }
+
+  @Override
+  public boolean isValid(FwmtCancelActionInstruction rmRequest, GatewayCache cache) {
```

Comment:
With the CeCancelUnitProcessor, the cache cannot be null, but with the EmptyCreateUnitCancel, the cache has to be null. 

I've had to have these here as if we are receiving an update or cancel before a create has taken place, the current rules are expecting there to be a record for them otherwise it will fail. It was the only way I could see to code it without adding additional processor keys and/or majorly editing the JobService class.